.github/workflows/integration-testing.yml

```
@@ -0,0 +1,194 @@
+name: Integration Testing
+on: push
+
+jobs:
+  build_Java_Agent:
+    name: Build AWS OTel Java agent
+    runs-on: ubuntu-latest
+
+    steps:
+      - uses: actions/checkout@v2
+
+      - uses: actions/setup-java@v1
+        with:
+          java-version: 14
```

```suggestion
          java-version: 15
```

Guess may as well use latest (unless it actually doesn't build for some reason)